compute/elastic-metal/how-to/connect-to-server.mdx

```diff
@@ -51,6 +52,35 @@ This page shows how to connect to your Scaleway Elastic Metal server via SSH. Th
 
 You are now connected to your Elastic Metal server.
 
+## How to connect to a Windows server using remote access 
+
+To connect to your Elastic Metal server running Windows, you need to download a remote access tool. For the sake of this tutorial, we use [Microsoft Remote Desktop](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12). 
+
+1. [Create an Elastic Metal server](/compute/elastic-metal/how-to/create-server/) and choose a Windows image to run on it.
+
+  <Message type="important">
+
+    - Windows images are not available for the Aluminium range of Elastic Metal servers.
+    - Make sure you save your password securely after creating your Elastic Metal.
+
+  </Message>
+
+2. Download Microsoft Remote Desktop and launch it on your computer. 
+
+3. Click **Add PC**. A configuration page displays. 
+
+  <Lightbox src="scaleway-configuration-page-microsoft-remote.webp" alt="" />
+
+4. Enter your Elastic Metal server's IP address in the **PC name** field and click **Add**. 
+
+5. Right click your newly added PC and select **Connect**.
+
+6. When prompted, enter `Administrateur` in the **Username** field and your password in the **Password** field. 
+
+7. Click **Continue**. You are redirected to the Windows Server Manager.
+
+You are now connected to your Elastic Metal server from Windows.
+
 ## How to connect from Windows
```

Comment:
I think I'd be tempted to have the order of things on this page be:

- How to connect from OSX (Mac) and Linux
- How to connect from Windows
- How to connect to an Elastic Metal server running Windows

...to me it makes a bit more sense that way, but not sure if it's just me?